qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml

```diff
@@ -380,6 +368,26 @@ OF THE POSSIBILITY OF SUCH DAMAGE.
     <module name="com.qulice.checkstyle.DiamondOperatorCheck"/>
     <module name="com.qulice.checkstyle.JavadocParameterOrderCheck"/>
     <module name="com.qulice.checkstyle.JavadocTagsCheck"/>
+    <module name="com.qulice.checkstyle.SingleLineCommentCheck">
+      <property name="format" value=" */\*[^*].*\*/ *"/>
+      <property name="message" value="This kind of comment is not allowed."/>
+      <property name="id" value="CStyleCommentNotAllowed"/>
+    </module>
+    <module name="com.qulice.checkstyle.SingleLineCommentCheck">
+      <property name="format" value=" */\*\* +[^A-Z\{ ][\W\s\S]*"/>
+      <property name="message" value="First sentence in a comment should start with a capital letter."/>
+      <property name="id" value="FirstLetterInCommentShouldBeCapital"/>
+    </module>
+    <module name="com.qulice.checkstyle.MultiLineCommentCheck">
+      <property name="format" value=" */\*\*\W+\* +[^A-Z\{ ][\W\s\S]*"/>
+      <property name="message" value="First sentence in a comment should start with a capital letter."/>
```

Comment:
@PeJetuz I would highly recommend you to change  this error message:
```
First sentence in a comment should start with a capital letter.
```
-> 

```
First sentence in a multiline comment should start with a capital letter.
```

It will greatly help to distinct these two cases. The same you can do in tests.

Reply:
@volodya-lombrozo Thanks! I've changed the messages so they should be more friendly now.